tool/cli.js

```diff
@@ -202,8 +203,28 @@ program
   .action(
     tryOrExit(async ({ args }) => {
       const { slug, locale } = args;
-      Document.validate(slug, locale);
-      console.log(chalk.green("✓ All seems fine"));
+      let okay = true;
+      const document = Document.findByURL(buildURL(locale, slug));
```

Comment:
I think we need to show a friendly error message when the document can't be found.